lib/neovim-plugin.nix

```
@@ -107,7 +108,7 @@ with lib;
         in
         mkIf cfg.enable (mkMerge [
           {
-            extraPlugins = [ cfg.package ] ++ extraPlugins;
+            extraPlugins = optional installPackage cfg.package ++ extraPlugins;
```

```suggestion
            extraPlugins = (optional installPackage cfg.package) ++ extraPlugins;
```

Probably useless here, but I would rather have them (or swap the order and have the `optional` in second).

---

Yeah, `++` and `//` have lower priority than the function call, hence why we have to do things like `(args // { override = "hi"; })` elsewhere.

I agree it's easy to forget though and your suggestions are more readable.

---

Done